lib/livebook/config.ex

```diff
@@ -616,7 +616,7 @@ defmodule Livebook.Config do
   @doc """
   Returns the current version of running Livebook.
   """
-  def app_version, do: Application.spec(:livebook, :vsn) |> List.to_string()
+  def app_version(), do: Mix.Project.config()[:version]
```

Comment:
This is going to fail inside a release because there is no Mix.

```suggestion
  @app_version Mix.Project.config()[:version]
  def app_version(), do: @app_version
```

Reply:
Ahh yes!!